regression/pages/lms/index.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+from edxapp_acceptance.pages.lms.index import IndexPage
+from regression.pages.lms import LMS_BASE_URL, LMS_PROTOCOL
+
+LOGIN_BASE_URL = '{}://{}'.format(LMS_PROTOCOL, 'edx-base-ficus-theme.raccoongang.com')
```

Comment:
`edx-base-ficus-theme.raccoongang.com` should be moved to a configuration.

Reply:
changed